dynamiqs/utils/utils.py

```diff
@@ -435,10 +435,10 @@ def expect(O: Tensor, x: Tensor) -> Tensor:
 
 
 def norm(x: Tensor) -> Tensor:
-    r"""Returns the norm of a ket, bra or a density matrix.
+    r"""Returns the norm of a ket, bra or density matrix.
 
-    For kets and bras, the returned norm is $\sqrt{\braket{\psi|\psi}}$. For density
-    matrices, it is $\tr{\rho}$.
+    For ket and bra, the returned norm is $\sqrt{\braket{\psi|\psi}}$. For a density
+    matrix, it is $\tr{\rho}$.
```

Comment:
I think I prefered the plural version. But as you wish.

Reply:
We use singular everywhere.